src/risk-modelling.md

```diff
@@ -10,33 +10,21 @@ To write the FMA for your project, follow the [FMA template](https://github.com/
-<br>**Low**: The feature doesn’t involve any components that can cause a significant incident.
-<br>**Medium**: A bug in the feature could lead to a temporary denial of service, or small loss of value across all users, or a large loss of value across a small amount of users.
-<br>**High**: Bugs in the feature could lead to denials of service lasting days or more, or a significant loss of assets.
-
-2. Determine a subjective complexity category by comparing your code to the examples below:
-<br>**Low:** Any code that is easy to explain to a non technical person, and easy reason about as a whole.
-<br>**Medium:** Code with several components that are each easy to reason about, or with a single feature that is complex.
-<br>**High:** Large codebase with several components, which are complex in their own right. Use of math, algorithms, architectural patterns, integration patterns or features that are novel or difficult to explain to a non technical person.
-
-3. Find the required audits for your risk and complexity in the table below. Read more about [audit types](https://www.notion.so/About-the-Audit-Process-1b9f153ee162805e8adcd2d50237c622?pvs=21).
-<br>The table was [calibrated to past audits](https://www.notion.so/Calibration-1bbf153ee16280d0a17adebee7f797e3?pvs=21), but it is a statement of minimums and you always have the option to execute more audits than specified.
-<br>Internal audits should be executed before external audits.
-<br>For internal audits we currently use the Coinbase Protocol Security Team. We also have a Spearbit retainer that can be used for internal audits. EVM Safety doesn’t do internal audits.
-<br>Instead of an internal audit, you can also upgrade to a solo external audit or external audit by an up-and-coming team. It is also possible to replace an internal with an external auditor that advises during design and implementation.
-<br>For high complexity features, contests are also an option. 
-
-|  | **Low Risk** | **Medium Risk** | **High Risk** |
-| --- | --- | --- | --- |
-| **Low Complexity** | Peer Review | Internal Audit | External Audit |
-| **Medium Complexity** | Internal Audit | External Audit | Internal Audit + External Audit |
-| **High Complexity** | External Audit | Internal Audit + External Audit | External Audit x2 |
-
-It is in the interest of the Tech Lead to accurately estimate the complexity and risk of the feature, with all the help provided. A major update to the FMAs in later stages of the SDLC or a High vulnerability found in the last audit will impact delivery times more than preparing for an audit or two from early stages.
+For context, please read [this framework from John Mardlin](https://gov.optimism.io/t/op-labs-audit-framework-when-to-get-external-security-review-and-how-to-prepare-for-it/6864). The process belows refines it.
+
+Do threat modelling of your feature, in doing so, do not consider audits as mitigations yet. When the threat modelling is complete, use the scale below:
```

Comment:
nit, threat modeling is optional (you can do an FMA instead), so we should tweak this section accordingly